Added PR Strategy reference (#61)
UlisesGascon/openssf-scorecard-monitor: README.md

## 🚀 Advance Tips

> ### Avoid commit directly in the branch and generate a PR 

Reviewer: ```suggestion
### Avoid committing directly to the branch and instead generate a PR
```

> If you have the OpenSSF Scorecard recommended branch protection rules in place, it will be impossible to commit and push directly to the `main` branch. An easy alternative is to extend the pipeline to generate a PR for you:

Reviewer: ```suggestion
If you have implemented the recommended branch protection rules from the OpenSSF Scorecard, committing and pushing directly to the `main` branch will be impossible. An easy alternative is to extend the pipeline to automatically generate a PR for you:
```
